util/crs-rules-check/rules-check.py

```diff
@@ -569,6 +569,29 @@ def check_pl_consistency(self):
                     severity  = None     # severity
                     has_nolog = False    # rule has nolog action
 
+def removecomments(data):
```

Comment:
Can you also add a `"""` comment on what this method is supposed to do? Removing comments where? In general, only inside a `Sec(Rule|Action)`? It's not clear...